.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:jest/recommended",
+        "prettier"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    "rules": {
```

Comment:
Also, please take a&nbsp;look at&nbsp;https://github.com/mdn/kumascript/pull/1033 for some of&nbsp;the&nbsp;rules I’d like to&nbsp;enforce.

Reply:
What I see in #1033 is mostly rules being disabled. But many of those don't need to be disabled since I'm using prettier for code formatting.

I don't have particularly strong feelings about eslint rules, so if you want to make specific arguments for or against specific rules let's talk about that after this lands.

Reply:
I disabled them to&nbsp;allow the&nbsp;build to&nbsp;pass (the&nbsp;rules in&nbsp;#1033 are all copied from Kuma, where they’re all&nbsp;enabled).